scopes/generator/generator/new.cmd.ts

```diff
@@ -28,18 +31,22 @@ export class NewCmd implements Command {
     [
       'a',
       'aspect <aspect-id>',
-      'aspect-id of the template. mandatory for non-core aspects. helpful for core aspects in case of a name collision',
+      "aspect-id of the template's owner, mandatory for non-core aspects. helpful for core aspects in case of a name collision",
     ],
-    ['t', 'template <env-id>', 'env-id of the template. alias for --env.'],
-    ['', 'env <env-id>', 'env-id of the template'],
-    ['d', 'default-scope <scope-name>', `set defaultScope in the new workspace.jsonc`],
+    ['t', 'template <env-id>', "env-id of the template's owner. Alias for --env."],
```

Comment:
what is the reference for "owner" here?
owner is a term in bit.cloud, not bit.

Reply:
same - see suggestion above